framework/db/src/main/java/com/cloud/utils/crypt/EncryptionSecretKeyChanger.java

```diff
@@ -497,6 +499,30 @@ private boolean migrateData(String oldDBKey, String newDBKey, String oldEncrypto
         return true;
     }
 
+    private void migrateEncryptedAccountDetails(Connection conn) {
+        System.out.println("Beginning migration of account_details encrypted values");
```

Comment:
Can you please remove this `System.out.println` line.

Reply:
@soreana this is useful for debugging when the `cloudstack-migrate-databases` is executed in the CLI. For example, this [comment](https://github.com/apache/cloudstack/pull/6812#issuecomment-1516808436) presents the output when using the command.